sklearn/externals/funcsigs.py

```diff
@@ -20,8 +20,8 @@
 __all__ = ['BoundArguments', 'Parameter', 'Signature', 'signature']
 
 
-_WrapperDescriptor = type(type.__call__)
```

Comment:
BTW I confirm that `type(bytearray.__add__)` returns `wrapper_descriptor` in CPython 2.7 (and in CPython 3.5 as well but we don't use this backport module under Python 3) so the proposed change seems harmless to me.